draft-ietf-httpbis-client-hints.md

```diff
@@ -247,10 +247,13 @@ If Downlink occurs in a message more than once, the minimum value should be used
 The "Save-Data" request header field is a token that indicates client's preference for reduced data usage, due to high transfer costs, slow connection speeds, or other reasons.
 
 ~~~ abnf7230
-  Save-Data = "on"
+  Save-Data = "Save-Data" ":" sd-token *( ";" [ sd-parameter ] )
+
+  sd-token = "on"
+  sd-parameter = token
 ~~~
 
-The token is a signal indicating explicit user opt-in into a reduced data usage mode on the client, and when communicated to origins allows them to deliver alternate content honoring such preference - e.g. smaller image and video resources, alternate markup, and so on.
+The "on" token is a signal indicating explicit user opt-in into a reduced data usage mode on the client, and when communicated to origins allows them to deliver alternate content honoring such preference - e.g. smaller image and video resources, alternate markup, and so on.
 
```

Comment:
You need to specify how future extensions are defined. The easiest thing to do is to say that updates to this specification can define more; we can always establish a registry later.